src/Raven.Client/Documents/Operations/CompareExchange/DeleteCompareExchangeValueOperation.cs

```diff
@@ -7,11 +7,24 @@
 
 namespace Raven.Client.Documents.Operations.CompareExchange
 {
+    /// <summary>
+    /// Operation to delete a compare exchange value in a RavenDB database. 
```

Comment:
I would remove the `in a RavenDB database` in all of those places..

Reply:
Done